src/shoobx/immutable/tests/test_pjpersist.py

```diff
@@ -224,14 +224,24 @@ def test_load_one(self):
 
     def test_add(self):
         im = pjpersist.Immutable()
+
+        # can NOT add a transient object
+        with self.assertRaises(AssertionError):
+            self.cont.add(im)
```

Comment:
Please make the error case its own test method called `test_add_withTransientObject`.